server/src/utilities/adminInit.ts

```diff
@@ -0,0 +1,12 @@
+const admin = require('firebase-admin');
+const serviceAccount = require("../../src/private_key/private.json");
```

Comment:
What decision led to this being in a new file & directory instead of being added to the env?

Reply:
This is because our service client credentials are inside a JSON file instead of being api keys.

For me, the .env file is good for variables we need to use but the firebase-admin initializeApp takes in a whole JSON file to set up!